src/docfx/build/xref/XrefMap.cs

```diff
@@ -55,16 +55,15 @@ public void OutputXrefMap(Context context)
         }
 
         private (Error error, string href, string display, IXrefSpec xrefSpec) ResolveCore(
-            string uid, SourceInfo<string> href, string displayPropertyName, Document relativeTo, string moniker = null)
```

Comment:
rebase add a test

Reply:
https://github.com/dotnet/docfx/blob/f8a8fca15ea624d1f53f8c773aacaf403220dfc1/docs/specs/xref.yml#L488
We have a test of `Markdown inclusion with xref reference`, the resolved URL is a relative path